third_party/move/move-model/src/model.rs

```diff
@@ -2634,6 +2633,13 @@ impl<'env> ModuleEnv<'env> {
         self.env.file_id_is_primary_target.contains(&file_id)
     }
 
+    /// Returns true if both modules are defined in the same package.
+    pub fn in_same_package(&self, other: &'env Self) -> bool {
```

Comment:
Submitted issue https://github.com/aptos-labs/aptos-core/issues/13745 for this.

Reply:
As said above, I think we only need to distinguish whether a module is part of the currently compiled package or not. And this is defined by `is_primary_target` (`is_target` was recently renamed as such, which was an unfortunate decision because it already created a lot of confusion).